configurations/default/env.yml.tmp

```diff
@@ -1,11 +1,12 @@
-AUTH0_DOMAIN: your_domain.auth0.com
```

Comment:
@landonreed we still need these auth0 config params, don't we?

Reply:
@br648 We still need `AUTH0_DOMAIN`, `AUTH0_API_CLIENT`, and `AUTH0_API_SECRET`. Requests are not going through without.